src/renderer/mod.rs

```diff
@@ -1,6 +1,9 @@
-use crate::display_list::{DisplayList, Margin};
+pub mod margin;
+
+use crate::display_list::DisplayList;
 use crate::snippet::Snippet;
 use anstyle::{AnsiColor, Effects, Style};
+use margin::Margin;
```

Comment:
We should `mod margin` and `pub use margin::Margin`

There is little gained from deeply nesting the API like this